src/connector/src/kinesis/source/reader.rs

```diff
@@ -41,7 +41,7 @@ pub struct KinesisSplitReader {
 }
 
 #[async_trait]
-impl SplitReader for KinesisSplitReader {
+impl SourceReader for KinesisSplitReader {
```

Comment:
SourceReader is ambiguous with the upper-level abstraction StreamSourceReader. SplitReader sounds more low-level and actually it's more aligned with the name of KinesisSplitReader and KafkaSplitReader.